actionpack/lib/action_dispatch/system_testing/test_helpers/screenshot_helper.rb

```diff
@@ -62,7 +62,7 @@ def unique
           end
 
           def image_name
-            sanitized_method_name = method_name.tr("/\\", "--")
+            sanitized_method_name = method_name.gsub(/[^\w]+/, "-")
```

Comment:
While we do more [complex sanitation](https://github.com/rails/rails/blob/69296dd7162d3524ce88ab1476896ad7474d4e84/activestorage/app/models/active_storage/filename.rb#L58) in other places, this seems like a clean and simple solution for method names. Only allowing letters, numbers and underscores should still create recognizable image name, while the unique token will make sure there won't be any overlap.

Reply:
Well, on paper `gsub(/[^\w]+/, "-")` allow for clashes, eg. `foo....` and `foo.` are both mangled into `foo-`. But I don't think it's a big deal, and an easy fix would be to remove the `+`.